reactjs-todo/e2e/centralized-login.spec.js

```diff
@@ -0,0 +1,19 @@
+import { test, expect } from '@playwright/test';
+
+test('centralized login react', async ({ page }) => {
+  await page.goto('https://localhost:8443/?centralLogin=true', { waitUntil: 'networkidle' });
```

Comment:
lets look for alternatives to network idle

Reply:
set to default and worked(load)

Reply:
I set the 'waitUntil' option to 'load' which is the default option and worked as expected.